src/components/TheFooter.vue

```diff
@@ -86,7 +86,7 @@
           <a
             href="https://www.linkedin.com/in/muhammadhafijur/"
             target="_blank"
-            class="text-gray-600 dark:text-blue-700 transition duration-300 ease-out hover:scale-110 hover:text-blue-700"
+            class="text-gray-600 dark:text-slate-50 transition duration-300 ease-out hover:scale-110 hover:!text-blue-700"
```

Comment:
Using `hover:!text-blue-700` is not ideal because `!` (important) can make the code harder to maintain and override. Instead, you can achieve the same effect by using `dark:hover:text-blue-700`. This approach keeps the code cleaner and more maintainable, avoiding the need for `!`important.

Reply:
> Using `hover:!text-blue-700` is not ideal because `!` (important) can make the code harder to maintain and override. Instead, you can achieve the same effect by using `dark:hover:text-blue-700`. This approach keeps the code cleaner and more maintainable, avoiding the need for `!`important.

thanks for clarification , done the changes